crates/eframe/src/web/events.rs

```diff
@@ -564,6 +564,11 @@ fn install_touchend(runner_ref: &WebRunner, target: &EventTarget) -> Result<(),
                 runner.needs_repaint.repaint_asap();
                 event.stop_propagation();
                 event.prevent_default();
+
+                if runner.text_agent.has_focus() {
+                    runner.text_agent.set_focus(false);
+                    runner.text_agent.set_focus(true);
+                }
```

Comment:
What the intention to do that? If this is some workaround it should be documented explicitly, because it could introduce unintended behaviour in other places and if such behaviour would be found and traced down to this lines is should be obvious why they are here.